### ISSUE TITLE
valami enhancement a képzésekről

### ISSUE THREAD

**Issue by zilahir** [arphox/tudasbazis#22]
Legalább heti egyszer előjönnek [ilyen](https://www.facebook.com/groups/408156912634809/permalink/2365211026929378/) posztok. 

Kellene valami konszenzus, hogy hogyan kezeljük ezeket, milyen irányba tereljük a kérdezőt. 

Én valami olyasmire gondolok első sorban (nyilván azontúl, hogy a _normális_nak tűnő képzéseket felsoroljuk, hogy egy set of questions: _Mit kérdezz meg magadtól mielőtt belevágnál_ jellegű megközelítés, mert az az érzésem, hogy a legtöbben valami egész más okból gondolkoznak el ezen a kérdéscsoporton, mint kellene. 

Mit gondoltok?

**Comment by arphox** [comment]
Nekem tetszik az ötlet, habár nem tudom hogy egyáltalán hányan használják ezt a tudásbázist. 😄 
(Kicsit fel kellene megint pörgetni a dolgot, legalább az alap cikkeket befejezni.)

És hogyan gondolnád ezt használni?
Csinálunk erről egy leírást, és ha valaki ilyet kérdez, belinkeljük ezt a cikket?

**Comment by ghost** [comment]
Vagy az alap szabályzatba beleírjuk, és amikor ilyet kérdez, akkor meg lehet adni, hogy miért törlöd a posztot. Ha ezt a szabályzatot kiválasztod, akkor látja, hogy "jaaaaa, már van ilyen"

**Comment by zilahir** [comment]
@arphox gondolkoztam a kérdésen, talán egyel mehetnénk tovább is ezügyben, egy komplexes leírással oktatás kapcsán, kezdve a hazai felsőoktatássi intézmények által nyújtott szakoktól, az OKJ-s képzéseken át, a codecampekig. 

A csoportban van  sok példa mindegyik csoportból, akár ők is megírhatnák 1-1 `pr`-ben amit a saját intézményéről gondol. 

De igen, a csoportból ebben a témában keletkezett posztokat is összelehetne gyűjteni. Valami ilyesmit képzeltem el elsőre.

**Comment by arphox** [comment]
Igen, viszont ezek részben nagyon szubjektív dolgok; elvárásoktól függően valaki egy képzést nagyon leszólhat vagy isteníthet. Ettől függetlenül jó ötletnek tartom, ha megpróbálunk minél inkább rámenni az objektív tartalomra, pl. milyen tantárgyak vannak, nagyjából miket tanítanak meg (tematika), mik a követelmények, stb.

**Comment by zilahir** [comment]
igaz-igaz,  bár a követelményeket  nehéz lenne up-do-date tartani, az azért tud gyakran változni, de ja, valami jó volna mindenképp.
